fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraTombstones.java

```diff
@@ -68,7 +68,7 @@ public FedoraTombstones(final String externalPath) {
     public Response delete() {
         LOGGER.info("Delete tombstone: {}", resource());
         resource().delete();
-        session.commit();
+        transaction.commit();
```

Comment:
I think deleting tombstone should still only commit on explicit commit by the client, unless this is a short lived transaction.

Reply:
Done